### ISSUE TITLE
Modeller Arguments are not passed

### ISSUE THREAD

**Issue by thempel** [markovmodel/adaptivemd#17]
If I'm not somehow missing something, the arguments that can be given to the `PyEMMAAnalysis` class (`.args`) are simply stored, but not passed. Even though I kind of like it (no more error messages...), this should somehow work. Best would be to make it congruent with the arguments passed to `OpenMMEngine`.

**Comment by jhprinz** [comment]
Actually, yes. The args argument is not used at all. I will change this to work like this

```py
task = modeller.task_msm_model(list(project.trajectories), tica_dims = 3, msm_lagtime = 50, ...)
```

**Comment by thempel** [comment]
Great, thanks!

**Comment by thempel** [comment]
Fixed by #21
